src/Appwrite/Platform/Modules/Sites/Http/Sites/CreateSite.php

```diff
@@ -66,7 +66,7 @@ public function __construct()
             ->param('outputDirectory', '', new Text(8192, 0), 'Output Directory for site.', true)
             ->param('subdomain', '', new CustomId(), 'Unique custom sub-domain. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.', true)
             ->param('buildRuntime', '', new WhiteList(array_keys(Config::getParam('runtimes')), true), 'Runtime to use during build step.')
-            ->param('serveRuntime', '', new WhiteList(array_keys(Config::getParam('runtimes')), true), 'Runtime to use when serving site.')
+            ->param('adapter', '', new Text(8192, 0), 'Framework adapter. Usuallly allows: static, ssr')
```

Comment:
Should it be a whitelist validator instead?

Reply:
We cant have whitelist yet, the whitelist values are only defined after we know framework. I added validator first thing inside the action

Reply:
I opened disucssion with team leads, Ill update it if possible in-line. But probably not, so this is good enough.